### ISSUE TITLE
[Suggestion] Snapshots & Raider Upgrades

### ISSUE THREAD

**Issue by LastTourniquet** [Heishe/idleraiders-second-run#549]
Allow Snapshots to save Raider Upgrade selections. Some Raider Upgrades are better for farming and some are better for raiding, it would be convenient if snapshots could save these along with the items/abilities that we set for each character.

**Comment by Jubetsu90** [comment]
This goes well with my automatic snapshot swap-suggestion. When going to raid - put on raid snapshot and raider upgrades. When farming - put on farming gear and raider upgrades. Making it automatic would allow raiding and farming in idle without keeping raiding gear and upgrades on while farming

**Comment by LastTourniquet** [comment]
@Jubetsu90 I don't think it should be automatic. I am aware the game is tagged as Idle but (at least for me) one of the fun parts of the game is making sure my characters are wearing the rite gear at the rite time. If there was an automatic swap whenever  you simply stopped raiding it would get rid of  a large chunk of the interactions you have with the game and drastically reduce the enjoyment (or at least my enjoyment) of the game.
That being said I don't see why Snapshots don't already snapshot the entire board and only snapshot the gear/abilities. It is strange to me that raider upgrades aren't inherently part of this by default. Whats even more worrying is that if you are unaware of this and you assume it does change your Raider Upgrades then if you try to change to Farming and have (for example) Hail of Arrows **only** on your Farming setup it won't change any of the gear on any of your raiders since, as far as the snapshot knows, you are **missing** those skillbooks. When infact they are just locked behind an activate button.

**Comment by Jubetsu90** [comment]
are you saying you enjoy swapping snapshots every hour to raid gear and back? because i don't do that, instead i auto-raid whatever i can in my ridiculous xp-gear. And yeah the game is called "Idle" Raiders so it kinda suggests you don't need to be watching your gear every damn time you auto-raid and start farming again

**Comment by LastTourniquet** [comment]
@Jubetsu90 Actually. Yes, I do enjoy swapping my gear out and checking to see if I have any improvements for my gear every time I start raiding. It seems like you want a game that will literally play itself for you, which isn't something that I personally enjoy. Yes it has the **idle** tag on it but so does Idle Clicker, or Adventure Capitalist, both of which are games that require you to actually sit there and play the game to make any real sense of progress, and both of which are very successful **idle** games.

My suggestion was simply to tie builds to Raider Upgrades because it seems like an obvious thing to do to me, not to make the game more automated. If I gave off the wrong impression I am sorry.

**Comment by Jubetsu90** [comment]
I didn't mean automatic raid gear would automatically calculate something or optimize. I mean you have made a snapshot which would be turned on when raiders take off to auto-raiding. Then turn back to xp-swords and aoe snapshot you have made when they come back. Just in case you had misunderstood me.

The progress isn't in swapping snapshots between these two. Progress and fun is when you add new items and skills to your snapshots, try gaining higher farming levels, try winning new raids... etc. The actual swapping of gear every 30 min is what annoys me. If you don't swap, you must choose between decent xp farming and decent auto-raiding

**Comment by Falopi** [comment]
I think just making snapshots overwritable would make things a lot smoother with the snapshot feature.
I'm with LastTourniquet, we don't need an automatic gear equipping system, that takes the fun out of the little interaction we do have.

**Comment by Falopi** [comment]
@Jubetsu90 and that SHOULD be a choice you have to make before idling for HOURS on end. Do i wanna farm raids, or do i wanna farm exp/global drops?

**Comment by Jubetsu90** [comment]
yeah well who would choose auto-raiding since you're not going to survive the highest level raids on auto anyway. That leaves auto-raiding only for the low level crap nobody needs, thus rendering auto-raiding basically useless

**Comment by Falopi** [comment]
soooo what is the point you are trying to make >.<

**Comment by Jubetsu90** [comment]
auto-raiding is pretty useless without being able to gear for it automatically too

**Comment by Jubetsu90** [comment]
do you disagree or just dislike the fact? Cause it's not my fault
